v3-sdk/trading/src/config.ts

```diff
@@ -43,8 +44,8 @@ export const CurrentConfig: ExampleConfig = {
   },
   tokens: {
     in: WETH_TOKEN,
-    amountIn: 0.1,
+    amountIn: 1,
     out: USDC_TOKEN,
-    poolFee: FeeAmount.MEDIUM,
+    poolFee: FeeAmount.LOW,
```

Comment:
Any reason for these changes specifically?

Reply:
Not necessarily, I can roll back